pkg/server/backuprestoreserver.go

```diff
@@ -297,6 +298,7 @@ func (b *BackupRestoreServer) runServer(ctx context.Context, restoreOpts *brtype
 			if restoreOpts.OriginalClusterSize > 1 {
 				m := member.NewMemberControl(b.config.EtcdConnectionConfig)
 				if err := m.PromoteMember(ctx); err == nil {
+					metrics.IsLearner.With(prometheus.Labels{}).Set(0)
```

Comment:
Also set `HasLearner` to false, since the learner was promoted to member here.

Reply:
Can you rather set this to `false` within `PromoteMember` of `member_control.go` so that we set the metric in all cases of member promotion?